modules/ROOT/pages/exchange-api.adoc

```diff
@@ -149,6 +153,37 @@ If the asset is published asynchronously, then a response like this is returned:
 
 The endpoint `publicationStatusLink` provides the asset publication status.
 
+[[set-mutable-data]]
+=== Set Mutable Data
+
+This example shows how to create your asset and set the mutable data describing it in the same request. The mutable data of an asset includes categories, custom fields, tags, and documentation pages.
+
+This example publishes the category and custom field, which have the keys `categoryKey` and `fieldKey`. They must be configured for your organization. Publication fails if they are not configured for your organization or if they are not deleted from the pom.
+
+It is not possible to submit new mutable data to an existing patch version such as 1.2.3. Instead, create a new patch version such as 1.2.4 with updated mutable data such as categories.
+
+The documentation archive file sent as `files.docs.zip` contains each page as a Markdown file such as `Getting Started.md` in the root directory of the archive. Replace `Getting Started` with the title of the page. Include resources such as images in the optional subdirectory `resources`. Specify the page order with the property `pageOrder` on the root of the main object in the optional file `config.json`:
```

Comment:
```suggestion
The documentation archive file sent as `files.docs.zip` contains each page as a Markdown file such as `Getting Started.md` in the root directory of the archive. Replace `Getting Started` with the title of the page. Include resources such as images in the optional subdirectory `resources`. To modify the terms and conditions page, the file `terms.md` must be present and its content will be the content of that section. Specify the page order with the property `pageOrder` on the root of the main object in the optional file `config.json`:
```
Could we add the `terms.md` section here? ☝️

Reply:
That's perfect! Added.